src/main/kotlin/org/wa/rceditor/application/Styles.kt

```diff
@@ -83,12 +109,25 @@ class Styles : Stylesheet() {
         }
         addItemRoot {
             padding = box(.5.em)
-            textField {
-                prefWidth = 200.px
-            }
         }
         boldLabel {
             fontWeight = FontWeight.BOLD
         }
+        tabHeaderBackground {
+            backgroundColor += c("#94008B")
+        }
+        dialogBody {
+            padding = box(10.px)
+            prefWidth = 500.px
+
+            label {
+                textFill = c("#000")
+            }
+        }
+        /*listCell {
+            and(selected) {
+                backgroundColor += c("#94008B")
+            }
+        }*/
```

Comment:
Remove commented code?

Reply:
@mbr4477 I will remove it, I'm still playing with styles